glen-anderson/msdk
# feat(Documentation): Update theme to adi-harmonic-mkdocs

### Description

- Update the theme to use [adi-mkdocs-harmonic](https://pypi.org/project/adi-mkdocs-harmonic)
- Add a few missing entries to `.gitignore`
- Update `CONTRIBUTING.md` with advice on creating a virtual environment for building the docs

### Testing

To test:

```sh
pip install -r Documentation/requirements.txt
mkdocs server
```

You should see the new theme in your browser

### Checklist Before Requesting Review

- [x] PR Title follows correct guidelines.
- [x] Description of changes and all other relevant information.
- [ ] (Optional) Link any related GitHub issues [using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [ ] (Optional) Provide info on any relevant functional testing/validation.  For API changes or significant features, this is not optional.
